src/Compilers/VisualBasic/Portable/Symbols/EventSymbol.vb

```diff
@@ -137,6 +137,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Private ReadOnly Property IEventSymbol_Parameters As ImmutableArray(Of IParameterSymbol) Implements IEventSymbol.Parameters
+            Get
+                Return ImmutableArray(Of IParameterSymbol).CastUp(Me.DelegateParameters)
+            End Get
+        End Property
```

Comment:
actually, i leave it to you guys to tell me if you want this.  Would you prefer this to sit next to the 'DelegateParameters' property, or do you want it in the IEventSymbol section?  I'll do whatever you guys prefer.